lib/vanagon/platform.rb

```diff
@@ -271,5 +272,13 @@ def is_cross_compiled?
     def is_cross_compiled_linux?
       return (is_cross_compiled? && is_linux?)
     end
+
+    # Utility matcher to determine if the platform uses host OS provided build tools
+    # (toolchain, boost, etc).
+    #
+    # @return [true, false] true if the platform uses host OS build tools, false otherwise
+    def uses_vendor_build_tools?
+      return @vendor_build_tools
```

Comment:
Ditch the explicit `return`, and consider always coercing this value into a pure boolean. Many DSL parameters are passed as strings, and the existing DSL method does no type checking for what is passed into it.